assets/js/components/ChargingSessionModal.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+	<Teleport to="body">
+		<div
+			id="sessionDetailsModal"
+			class="modal fade text-dark"
+			tabindex="-1"
+			role="dialog"
+			aria-hidden="true"
+		>
+			<div class="modal-dialog modal-dialog-centered modal-dialog-scrollable" role="document">
+				<div v-if="session != undefined" class="modal-content">
+					<div class="modal-header">
+						<h5>{{ $t("session.title") }}</h5>
+						<button
+							type="button"
+							class="btn-close"
+							data-bs-dismiss="modal"
+							aria-label="Close"
+						></button>
+					</div>
+					<div class="modal-body">
+						<table class="table">
+							<tbody>
+								<tr>
+									<th colspan="2"></th>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.csv.created") }}
+									</th>
+									<td>
+										{{ fmtFullDateTime(new Date(session.created), false) }}
+									</td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.csv.finished") }}
+									</th>
+									<td>
+										{{ fmtFullDateTime(new Date(session.finished), false) }}
+									</td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.energy") }}
+									</th>
+									<td>
+										{{ fmtKWh(session.chargedEnergy * 1e3) }}
+									</td>
+								</tr>
+								<tr>
+									<th></th>
+									<td></td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("main.loadpoint.fallbackName") }}
+									</th>
+									<td>
+										{{ session.loadpoint }}
+									</td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("session.meterstart") }}
+									</th>
+									<td>
+										{{ fmtKWh(session.meterStart * 1e3) }}
+									</td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("session.meterstop") }}
+									</th>
+									<td>
+										{{ fmtKWh(session.meterStop * 1e3) }}
+									</td>
+								</tr>
+								<tr>
+									<th colspan="2"></th>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.vehicle") }}
+									</th>
+									<td>
+										{{ session.vehicle }}
+									</td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("session.odometer") }}
+									</th>
+									<td>{{ session.odometer }} km</td>
```

Comment:
We need to use the `distanceValue` and `distanceUnit` helpers here to support km/miles switching.

See here: https://github.com/evcc-io/evcc/blob/master/assets/js/components/Vehicle.vue#L22